config/initializers/patches.rb

```diff
@@ -1,3 +1,5 @@
 Dir[Rails.root.join("lib/patches/*.rb")].sort.each do |patch|
   require patch
 end
+
+Rails.autoloaders.main.ignore("lib/patches")
```

Comment:
It'd be good to add either a comment above this or information to the commit message to just state that we don't files in lib/patches to follow standard Zeitwerk conventions and they should all be required manually.